ext/http/00_serve.ts

```diff
@@ -658,22 +660,11 @@ function serve(arg1, arg2) {
     loadBalanced: options[kLoadBalanced] ?? false,
   };
 
-  if (options.certFile || options.keyFile) {
-    throw new TypeError(
-      "Unsupported 'certFile' / 'keyFile' options provided: use 'cert' / 'key' instead.",
-    );
-  }
-  if (options.alpnProtocols) {
-    throw new TypeError(
-      "Unsupported 'alpnProtocols' option provided. 'h2' and 'http/1.1' are automatically supported.",
-    );
-  }
-
```

Comment:
Let's omit these changes for now. We can take of this in a separate PR, as part of removals of `certFile`, `keyFile` and `privateKey`.

Reply:
Ok, I won't bother updating them to conform to the new guidelines as that will lead to conflicts if they are already updated in a different MR.